examples/webgl_interactive_draggablecubes.html

```diff
@@ -149,17 +142,15 @@
 				mouse.x = ( event.clientX / window.innerWidth ) * 2 - 1;
 				mouse.y = - ( event.clientY / window.innerHeight ) * 2 + 1;
 
-				//
-
 				raycaster.setFromCamera( mouse, camera );
 
 				if ( SELECTED ) {
 
-					var intersects = raycaster.intersectObject( plane );
+					var intersection = plane.intersectRay(raycaster.ray);
```

Comment:
You need to make use of the `optionalTarget` to avoid instantiating `Vector3`'s unnecessarily.